fetch_gazebo_demo/scripts/demo.py

```diff
@@ -149,7 +149,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
```

Comment:
Since this API has already changed once, I would suggest that we keep this as a kwarg, `use_service=True`

Reply:
Thanks!
I updated the code.

Reply:
Note: I merged a fix commit into this branch to change to use_service=False as recommended by Mike Ferguson.